roles/idm-host-cert/README.md

```diff
@@ -0,0 +1,100 @@
+Set of Roles
+============
+
+The ansible roles found in this directory are associated with creating and configuring idm host certs.
```

Comment:
I'd update this to read:
`This ansible role is used to create certificates, using the Red Hat IdM as a Certificate Authority`